samples/sample-project/build.gradle.kts

```diff
@@ -0,0 +1,162 @@
+        packageContext =
+            AvailLibraryPackageContext("myJar", "$buildDir/libs").apply {
+                // Add any key-value pairs to the manifest included in the jar.
+                manifestPairs["some-key"] = "some-value"
+                // An action that will happen after the jar file is created.
+                postPackageAction = {
+                    println(
+                        "Hi there, this is where the file is: ${it.absolutePath}")
+                }
+            }
+        // Add a module package to this created root. Only happens if file does
+        // not exist.
+        modulePackage("App").apply{
+            // Specify module header for package representative.
+            versions = listOf("Avail-1.6.0")
+            extends = listOf("Avail", "Configurations", "Network")
+            // Add a module to this module package.
+            addModule("Configurations").apply {
+                // Specify module header for this module..
+                versions = listOf("Avail-1.6.0")
+                uses = listOf("Avail")
+                moduleHeaderCommentBody = customHeader
+            }
+            // Add a module package to this module package.
+            addModulePackage("Network").apply {
+                println("Settingup Network.avail")
+                versions = listOf("Avail-1.6.0")
+                uses = listOf("Avail")
+                extends = listOf("Server")
+                moduleHeaderCommentBody = customHeader
+                addModule("Server").apply {
+                    versions = listOf("Avail-1.6.0")
+                    uses = listOf("Avail")
+                    moduleHeaderCommentBody = customHeader
+                }
+            }
+        }
+
+        // Add a module to the top level of the created root.
+        module("Scripts").apply {
+            versions = listOf("Avail-1.6.0")
+            uses = listOf("Avail")
+            moduleHeaderCommentBody = customHeader
+        }
+    }
+}
+
+tasks {
+    // Customize task that runs default workbench.
+    assembleAndRunWorkbench {
+        // This task is customizable in the same manner as any
+        // AvailWorkbenchTask.
+        dependency("org.slf4j:slf4j-nop:2.0.0-alpha5")
+    }
+
+    // Add your own custom task to assemble and launch an Avail workbench.
+    val myWorkbenchTask by creating(AvailWorkbenchTask::class.java)
+    {
+        group = "My Tasks"
+        description = "My custom workbench build."
+        dependsOn(jar)
+        workbenchJarBaseName = "myCustomWorkBench"
+        rebuildWorkbenchJar = true
+        maximumJavaHeap = "6g"
+        workbenchLocalJarDependency("$buildDir/libs/sample-project.jar")
+        dependency("org.slf4j:slf4j-nop:2.0.0-alpha5")
+        root("my-avail-root", "$projectDir/avail/roots/my-avail-root")
+        root(
+            "avail",
+            "jar:$projectDir/avail/roots/avail-stdlib-1.6.0.20210910.181950.jar")
```

Comment:
Isolate the version string, as an example of good practice for Avail users.
[I believe it's in Versions.availStripeVersion]